helm/alfresco-content-services/values.yaml

```diff
@@ -274,6 +275,9 @@ alfresco-transform-service:
       data:
         mountPath: "/tmp/Alfresco"
         subPath: "alfresco-content-services/filestore-data"
+    strategy:
+      # -- Strategy must be set to Recreate when persistence is enabled
+      type: Recreate
```

Comment:
not sure this is the best place where to set this given that it really depends on the `accessMode` of the `storageClass`